Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -204,3 +204,17 @@ the size of the cluster.
     ceph orch set backend ""
     ceph mgr module disable rook
     ```
+
+## External Clusters
+
+User Applications can make use of the NFS client running on the external ceph standalone cluster.
```

Comment:
Update wording to reflect my other "client", "Ceph" suggestions.

Also, "Applications" doesn't need to be capitalized.